File: CB.EN.U4CYS22057/30_03_2023/c/largest_in_an_array.c
#include <stdio.h>

int main() {
    // Declaring array
    int array[5] = {0};

    // Getting the values and assigning them in the array
    for (int x = 0; x < 5; x++) {
        printf("Enter %d value:\n", x+1);
        int input;
        scanf("%d", &input);

        // Assigning values in the array
        array[x] = input;
    }

    // Assigning the max value as the first value
    int max = array[0];

    // Checking the remaining values to see if they are greater than the first value
    for (int x = 1; x < 5; x++) {
        if (array[x] > max) {
            // Updating the max value to the largest value found
            max = array[x];
        }
    }

    // Outputting the largest value
    printf("The largest integer value in array is %d\n", max);

    return 0;
}

File: CB.EN.U4CYS22057/30_03_2023/python/largest_in_an_array.py
# Declaring array
array = [0] * (5)

# Declaring other variables
# Getting the values and assigning them in the array
for x in range(0, 4 + 1, 1):
    print("Enter " + str(x + 1) + " value")
    input = int(input())
    array[x] = input

# assigning the the max value as the first value
max = array[0]

# checking the remaing values to see if they are greater than the first value
for x in range(0, 4 + 1, 1):
    if array[x] > max:
        
        # updating the max value to the largest value found
        max = array[x]

# outputing the largest value
print("The largest integer value in array is " + str(max))
